### PR TITLE
update Notion-Version to 2022-02-22 

### PR DESCRIPTION
**Pull request by hkob**
Hello, thank you very much for developing the gem for Notion's API.
I use your `notion-ruby-client` library in my notion-ruby-mapping project.
I want to update my some PropertyClass object for Notion-Version 2022-02-22.

Your implementation code is almost unaffected by this change, but some of the test code needed to be changed.
So I changed API responses (*.yml) and corresponding RSpec verification values.
Moreover, List database endpoint was deprecated from Notion-Version 2022-02-22, so I removed the corresponding implementation and RSpec.

However, the "Retrieve a page property item" added this Notion-Version is not additionally implemented.

### CODE REVIEW

**Comment by phacks** [comment]
Thanks for your PR! I somehow missed the notification and just saw it. I’ll be reviewing it in the next few days!

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    notion-ruby-client (0.1.0.pre.beta1)
+    notion-ruby-client (0.1.0.pre.beta2)
```
Comment (phacks) [inline Gemfile.lock:4]: I’m not entirely sure why, but the test suite is failing at the `bundle install` step. Could you try and remove all changes from `Gemfile.lock` from your PR?

File: lib/notion/version.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 module Notion
-  VERSION = '0.1.0-beta1'
-  NOTION_REQUEST_VERSION = '2021-08-16'
+  VERSION = '0.1.0-beta2'
```
Comment (phacks) [inline lib/notion/version.rb:3]: Let’s not bump the version number directly in this PR—once it’s merged, I’ll create a separate commit to create `1.0.0-beta1`. I believe your PR is the last one to get this gem to 1.0.0!

**Review by phacks — COMMENTED** [review]
Thanks a lot for your contribution @hkob! The PR looks really good, I believe there are only a few minor changes needed before we can merge:

- [x] Remove changes from `Gemfile.lock` (if there are updates you _intentionally_ added there, please let me know!)
- [x] Remove the version update, this will be handled in a separate PR
- [ ] Update specs in the new endpoints I added yesterday (you might need to rebase on `main`:
  - Delete a block
  - Retrieve your token bot user
  - Retrieve a page property

Let me know if you need any help for the above, would be happy to assist!

**Comment by hkob** [comment]
I'm sorry that I forgot to fetch the original repository.
This patch was merged the head of the original repository.

**Pull request merged**